### DISCUSSION TITLE
Could I copy the migration files from 1 project to another and then migrate my pocketbase to update my collections?

### DISCUSSION THREAD

**Discussion by snorresovold** [pocketbase/pocketbase#1995] [Q&A] [answered]
I want to make a script/github action to automate migrating a pocketbase instance in prod from a file in a github repo, could I just copy the files or is there a better way to do this?

**Comment by ganigeorgiev** [comment]
I'm not entirely sure that I understand the question.

The `pb_migrations` directory can be safely copied to another environment and/or committed in your repo.

On `./pocketbase serve` all **unapplied** migrations from `pb_migrations` will be applied automatically.

**Comment by snorresovold** [comment]
I'm looking for the easiest way to get the migrations from 1 pocketbase instance to another, not any data just collections mainly.

I am currently trying to webscrape my pocketbase instance and use a scraper to login and input the json file for migrations, but I feel like that's a step in the wrong direction.

**Reply by ganigeorgiev** [reply]
> I'm looking for the easiest way to get the migrations from 1 pocketbase instance to another, not any data just collections mainly.

Just copy your `pb_migrations` directory.

---

> I am currently trying to webscrape my pocketbase instance and use a scraper to login and input the json file for migrations, but I feel like that's a step in the wrong direction.

I'm sorry, I don't understand what do you mean.

**Reply by snorresovold** [reply]
> I'm sorry, I don't understand what do you mean.

I'm using a Go script to go to my pocketbase admin site, write my login credentials, navigate to the migrate tab and upload my migration file.

**Reply by ganigeorgiev** [reply]
I'm still confused what you are trying to do. If you want to import collections via the web API, you can check the "Import collections" in https://pocketbase.io/docs/api-collections/. To get all collections, you can call "List collections".

But once again, if you already have a `pb_migrations` directory, copying this directory to the other environment will automatically "transfer" all collections structure changes to the new env.

**Reply by snorresovold** [reply]
Thank you, I'll try doing this then.

**Comment by snorresovold** [comment, marked as answer]
Seems like I found the solution to my problem

[here](https://pocketbase.io/docs/api-collections/)

This has an example of a way to migrate and change the schema with typescript. I'll make a github action and use that as an example
